pkg/controller/rest/verifiable/operation.go

```diff
@@ -240,3 +244,33 @@ func (o *Operation) GeneratePresentation(rw http.ResponseWriter, req *http.Reque
 func (o *Operation) GeneratePresentationByID(rw http.ResponseWriter, req *http.Request) {
 	rest.Execute(o.command.GeneratePresentationByID, rw, req.Body)
 }
+
+// RemoveCredentialByName swagger:route POST /verifiable/credential/remove/name/{name}
```

Comment:
this comment should be like this:
`RemoveCredentialByName swagger:route POST /verifiable/credential/remove/name/{name} verifiable RemoveCredentialByNameReg `
where `verifiable` groups your APIs by name `verifiable`
and `RemoveCredentialByNameReq` describes the request 

the swagger spec will be incorrect, you need to check it by `make run-openapi-demo` https://github.com/hyperledger/aries-framework-go/blob/master/docs/rest/openapi_demo.md

NOTE: it is related to every operation (models are missing)

Reply:
i'll add these